### PR TITLE
Introduce UnlockEquipmentRecipe MVP

### PR DESCRIPTION
**Pull request by ipdae**
- Unlock equipment recipe pay with CRYSTAL

### CODE REVIEW

File: Lib9c/Action/UnlockEquipmentRecipe.cs [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                unlockedIds.Add(recipeId);
+
+                EquipmentItemRecipeSheet.Row row = equipmentRecipeSheet[recipeId];
+                if (!worldInformation.IsStageCleared(row.UnlockStage))
+                {
+                    throw new NotEnoughClearedStageLevelException($"clear {row.UnlockStage} first.");
+                }
+
+                totalCost += 500;
+            }
```
Comment (Unengine) [inline Lib9c/Action/UnlockEquipmentRecipe.cs:85]: I think you should also throw exception when recipe which has lower `recipeId` exists.

**Comment by ipdae** [comment]
test

**Comment by ipdae** [comment]
test

**Comment by ipdae** [comment]
mic

**Review by sonohoshi — APPROVED** [review]

**Review by Unengine — APPROVED** [review]

**Pull request merged**